runtime/src/accounts.rs

```diff
@@ -606,6 +601,14 @@ impl Accounts {
         self.accounts_db.store(slot, &[(pubkey, account)]);
```

Comment:
nits: rename to `store_slow_uncached` or `store_slow_direct`?

Reply:
added, also a warning to say it's unsupported for use outside of tests/benches